### DISCUSSION TITLE
Quinoa serve only

### DISCUSSION THREAD

**Discussion by mschorsch** [quarkiverse/quarkus-quinoa#643] [Q&A] [answered]
Is it possible to configure Quinoa to disable building and live coding at all and to just serve content that was build by another build step (dev mode and production)?

**Comment by melloware** [comment]
so you have an already compiled JS application in your build folder and just want Quinoa to serve it?   Can you explain your use case in more detail?

**Reply by mschorsch** [reply]
> so you have an already compiled JS application in your build folder and just want Quinoa to serve it?

Yes.

> Can you explain your use case in more detail?

We build our js app in the CI in a separate build step, which is more efficient as we can create the modules of the quarkus application in parallel. The js app is one of these modules. Furthermore, the gradle build cache cannot be used with Quinoa because gradle cannot recognize which js app files were used for the build.

For live coding during development we simply use the nodejs proxy. So we don't need the build and live coding functionality of Quinoa but would like to use Quinoa to serve the js app especially because of the spa routing feature.

**Reply by melloware** [reply]
Interesting. I am not sure if it is possible @ia3andy will know more than myself.

**Reply by ia3andy** [reply]
When you say "build step", do you mean a Quarkus BuildStep or a CI build step?

**Reply by mschorsch** [reply]
I actually meant "Gradle Task", "build step" was the wrong term.

**Comment by mschorsch** [comment, marked as answer]
I've opened https://github.com/quarkiverse/quarkus-quinoa/issues/644
